websites/L/LambdaGeneration/presence.ts

```diff
@@ -0,0 +1,96 @@
+const presence = new Presence({
+	clientId: "1018020550095945828",
+});
+
+presence.on("UpdateData", async () => {
+	const splitted: string[] = window.location.pathname.substring(1).split("/"),
+		subComms: Record<string, string> = {
+			["half-life"]: "Half-Life",
+			["portal"]: "Portal",
+			["tf2"]: "Team Fortess 2",
+			["valve"]: "Valve",
+			["lambdageneration"]: "LambdaGeneration",
+			["black-mesa"]: "Black Mesa",
+			["sfm"]: "Source Film Maker",
+			["sven-co-op"]: "Sven Co-op",
+			["gmod"]: "Garry's Mod",
+			["csgo"]: "CS:GO",
+			["l4d"]: "Left 4 Dead",
+		},
+		pagee: string[] = ["Viewing", "trending"];
```

Comment:
I'm confused, why `pagee` not `page` or `pages`

Reply:
I thought `page` and `pages` was reserved